conans/test/unittests/util/tools_test.py

```diff
@@ -1119,7 +704,7 @@ def get_manual_auth(user, password):
         out = TestBufferConanOutput()
 
         # Connection error
-        with six.assertRaisesRegex(self, ConanException, "HTTPConnectionPool"):
+        with six.assertRaisesRegex(self, ConanException, "Error downloading"):
```

Comment:
Ok, I prefer to check for the `Error downloading file...` as it is the text we are writing in the `ConanException`. 

👍